### PR TITLE
Create new pages to integrate calendly's "get demo" integration #2710 

### LINKED ISSUE #2710: Create new pages to integrate calendly's "get demo" integration
**Context:**
We have a function to schedule a Demo ([https://www.gitpod.io/contact/sales?get-a-demo](https://www.gitpod.io/contact/sales?get-a-demo)), which, currently, deals with a lot of friction. It is not really easy to go from our website to talk to someone. This is why we decided to use a Calendly Routing Form to improve this process. The Routing form is created already: [https://calendly.com/d/d47-7n4-c63](https://calendly.com/d/d47-7n4-c63).

This can be embedded on the Gitpod website with the “Inline Embed” function in Calendly

**For this, we need to create two landing pages:**

1. `/get-demo` that has the demo booking form and potentially a little other useful information like link to screencasts
2. `/watch-demo` where people are routed to that should rather watch a demo, that shows the embedded video of the demo recording (it's basically a new screencast page with a little bit of extra information)

**Timeline:**
- create Mockup including review by @phimae until Tuesday EOD -> DRI: @chrifro 
- technical implementation including final review until Thursday -> DRI: @nisarhassan12 

View [Notion](https://www.notion.so/gitpod/Get-a-Demo-Function-Calendly-Routing-Form-6f87f17c564c418196372d90a49be5af) for more information

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #2710 

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2732"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2732"></a>

### CODE REVIEW

File: src/routes/screencasts/demo.svelte [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  );
+</script>
+
+<OpenGraph
+  data={{
+    title: "",
+    description: "",
+    keywords: "",
+  }}
+/>
```
Comment (nisarhassan12) [inline src/routes/screencasts/demo.svelte:30]: @jacobparis Please also suggest the value for these. Thanks

Comment (nisarhassan12) [inline src/routes/screencasts/demo.svelte:30]: @jacobparis Please also suggest values for these. Thanks

**Comment by nisarhassan12** [comment]
@chrifro the spacing around the embed on `/contact/get-demo` is not exactly as in the mockup. I have commented about it in the mockup. 
image

File: src/lib/components/get-demo/calendly-embed.svelte [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      @apply -mt-small;
+    }
+  }
+</style>
+
+<svelte:head>
+  <script
+    type="text/javascript"
+    src="https://assets.calendly.com/assets/external/widget.js"
+    async></script>
```
Comment (Siddhant-K-code) [inline src/lib/components/get-demo/calendly-embed.svelte:13]: Remove `async`. (Ref [this](https://help.calendly.com/hc/en-us/articles/360019861794-Common-embed-questions#1))

```suggestion
    src="https://assets.calendly.com/assets/external/widget.js"></script>
```

**Comment by chrifro** [comment]
Nice work @nisarhassan12 ✨
a few comments:

- [ ] switch to the option 2 layout for /get-demo as discussed on Slack [[1](https://gitpod.slack.com/archives/C01KGM9ETU6/p1663241217081549?thread_ts=1663078654.300589&cid=C01KGM9ETU6)]
- [ ] would it be possible to change the CTA font color for "Submit" to black within the calendly embed?
- [ ] update this screencast preview image to align with the YouTube Thumbnail [[1](https://www.figma.com/file/gq1v6dj0rzs4tvJcFabKPy/Blog-%26-Changelog-images?node-id=1342%3A825)]
Bildschirmfoto 2022-09-15 um 13 19 09

- [ ] align content for "helpful resources": 1) link to vs/local-development (for header image see mockup) 2) Blog post "cloud based development" 3) guide "gitpodifying" (for header image see mockup)
Bildschirmfoto 2022-09-15 um 13 20 43

- [ ] mobile: remove illustration
Bildschirmfoto 2022-09-15 um 13 21 21

File: src/lib/components/get-demo/calendly-embed.svelte [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      @apply -mt-small;
+    }
+  }
+</style>
+
+<svelte:head>
+  <script
+    type="text/javascript"
+    src="https://assets.calendly.com/assets/external/widget.js">
+</script>
```
Comment (Siddhant-K-code) [inline src/lib/components/get-demo/calendly-embed.svelte:13]: Fix calendly widget javascript src url

```suggestion
  <script
    type="text/javascript"
    src="https://calendly.com/assets/external/widget.js">
  </script>
```

**Review by Siddhant-K-code — COMMENTED** [review]
We also require a Title for the page

image

File: src/lib/components/get-demo/calendly-embed.svelte [hunk 4]
```diff
@@ -7,6 +7,7 @@
 </style>
 
 <svelte:head>
+  <title>Contact - Get a demo</title>
```
Comment (nisarhassan12) [inline src/lib/components/get-demo/calendly-embed.svelte:10]: Thanks, @Siddhant-K-code for looking into this is not the right way of doing it. I already asked Jacob in two comments above see https://github.com/gitpod-io/website/pull/2732#discussion_r971195844, https://github.com/gitpod-io/website/pull/2732#discussion_r971206921

Comment (nisarhassan12) [reply src/lib/components/get-demo/calendly-embed.svelte:10]: Please revert your change. Thanks

Comment (Siddhant-K-code) [reply src/lib/components/get-demo/calendly-embed.svelte]: Thanks for the context. I have reverted that change.

File: src/lib/components/get-demo/calendly-embed.svelte [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    async
+    src="https://calendly.com/assets/external/widget.js"
+    type="text/javascript">
+  </script>
+</svelte:head>
+
+<div
+  class="calendly-inline-widget"
+  data-url="https://calendly.com/d/d47-7n4-c63?primary_color=ffae33"
+  style="min-width:250px;height:650px;"
```
Comment (Siddhant-K-code) [inline src/lib/components/get-demo/calendly-embed.svelte:22]: We can change to following, to remove the scroll bars:

| Before | After  |
|--------|---|
| image        | image  |


```suggestion
  style="min-width:250px;height:726px;"
```

**Review by chrifro — COMMENTED** [review]
Looks great ✨

Two minor edits (after they have been addressed it can be merged directly)
- Link "get demo" from the sales form to /get-demo as well (see [Slack comment](https://gitpod.slack.com/archives/C01KGM9ETU6/p1663278560911909?thread_ts=1663078654.300589&cid=C01KGM9ETU6))
- move the whole text section of "get a demo" on the left side a bit further up. 
Bildschirmfoto 2022-09-16 um 08 19 10

File: src/routes/contact/get-demo.svelte [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  import Titles from "$lib/components/index/titles.svelte";
+  import UsedBy from "$lib/components/index/used-by.svelte";
+  import OpenGraph from "$lib/components/open-graph.svelte";
+</script>
+
+<OpenGraph
+  data={{
+    title: "",
+    description: "",
+    keywords: "",
```
Comment (jacobparis) [inline src/routes/contact/get-demo.svelte:17]:
```suggestion
    title: "Get a demo of Gitpod",
    description: "Supercharge your productivity by how you develop. Get a demo of Gitpod's continuous development platform to see how to maximize your developer experience.",
    keywords: "continuous development, remote development, demo, get started",
```

File: src/routes/screencasts/demo.svelte [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+      "cloud-based-development-for-everyone",
+    ].some((slug) => p.slug === slug)
+  );
+</script>
+
+<OpenGraph
+  data={{
+    title: "",
+    description: "",
+    keywords: "",
```
Comment (jacobparis) [inline src/routes/screencasts/demo.svelte:28]:
```suggestion
    title: "Gitpod Screencasts – Videos, guides, and tutorials",
    description: "Short screencasts to get started and become instantly ready to code. Set up, configure and customize Gitpod for continuous development in the cloud.",
    keywords: "learn, video, tutorial, documentation, onboarding ",
```

**Pull request merged**